src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGenerator.java

```diff
@@ -63,6 +64,20 @@ protected static String createRepoURI(final String url, final String projectName
         }
     }
 
+    /**
+     * Will for a given list of search criteria return the first found event
+     */
+    protected static String getPreviousEiffelEvent(final String linkedEiffelEventType, final String projectName,
+            final List<String> searchCriterias, final File pluginDirectoryPath) {
+        for (String searchCriteria : searchCriterias) {
```

Comment:
this can be final, and so is the eiffelEventId

Reply:
Fixed in 4d3fdb3